src/components/Footer.vue

```diff
@@ -1,12 +1,21 @@
+<style>
```

Comment:
It's a best practice in vue to write styles after script as it would be more readable

Reply:
write the styles below the script and make them `scoped`